### PR TITLE
layout CardContent to use screen real-estate effectively

### LINKED ISSUE #275: Layout could be improved for long chart names and version numbers
screen shot 2018-04-25 at 10 54 28 am

Two examples:
"elasticsearch-curator" is broken across two lines even through there is space in the tile. This might be because of the incubator label placement?

Dokuwiki is wrapped because of a very long version number. It seems like the chart name should take priority in this case? Since this looks like a corner case asking people to click through to see the exact version (or providing a tooltip for desktop users) seems reasonable.

### PR DESCRIPTION
**Pull request by sameersbn**
kubeapps dashboard

fixes #275

### CODE REVIEW

File: dashboard/src/components/AppList/AppListItem.tsx [hunk 1]
```diff
@@ -20,21 +20,22 @@ class AppListItem extends React.Component<IAppListItemProps> {
...
       <Card key={release.name} responsive={true} className="AppListItem">
-        <Link to={`/apps/ns/${release.namespace}/${release.name}`}>
+        <Link to={`/apps/ns/${release.namespace}/${release.name}`} title={release.name}>
           <CardIcon icon={iconSrc} />
           <CardContent>
             <div className="ChartListItem__content">
-              <h3 className="ChartListItem__content__title">{release.name}</h3>
-              <div className="ChartListItem__content__info text-r">
-                <p className="margin-reset type-color-light-blue">
+              <div className="ChartListItem__content__title type-big">{release.name}</div>
```
Comment (Angelmmiguel) [inline dashboard/src/components/AppList/AppListItem.tsx]: Please, keep it as a `h3`. It's important for the semantic web.

File: dashboard/src/components/AppList/AppListItem.tsx [hunk 2]
```diff
@@ -20,21 +20,22 @@ class AppListItem extends React.Component<IAppListItemProps> {
...
+              <div className="ChartListItem__content__info">
+                <div className="ChartListItem__content__info_version type-small padding-t-tiny type-color-light-blue">
```
Comment (Angelmmiguel) [inline dashboard/src/components/AppList/AppListItem.tsx]: Same here about the paragraph.

File: dashboard/src/components/ChartList/ChartListItem.tsx [hunk 3]
```diff
@@ -20,19 +20,22 @@ class ChartListItem extends React.Component<IChartListItemProps> {
...
           <CardIcon icon={iconSrc} />
           <CardContent>
             <div className="ChartListItem__content">
-              <h3 className="ChartListItem__content__title">{name}</h3>
-              <div className="ChartListItem__content__info text-r">
-                <p className="margin-reset type-color-light-blue">{latestAppVersion || "-"}</p>
-                <span
-                  className={`ChartListItem__content__repo ${repo.name} padding-tiny
-                  padding-h-normal type-small margin-t-small`}
+              <div className="ChartListItem__content__title type-big">{name}</div>
```
Comment (Angelmmiguel) [inline dashboard/src/components/ChartList/ChartListItem.tsx]: Same here

File: dashboard/src/components/ChartList/ChartListItem.tsx [hunk 4]
```diff
@@ -20,19 +20,22 @@ class ChartListItem extends React.Component<IChartListItemProps> {
...
+              <div className="ChartListItem__content__info">
+                <div className="ChartListItem__content__info_version type-small padding-t-tiny type-color-light-blue">
```
Comment (Angelmmiguel) [inline dashboard/src/components/ChartList/ChartListItem.tsx]: Same here

File: dashboard/src/components/FunctionList/FunctionListItem.tsx [hunk 5]
```diff
@@ -16,18 +16,25 @@ class FunctionListItem extends React.Component<IFunctionListItemProps> {
...
+        >
           <FunctionIcon runtime={f.spec.runtime} />
           <CardContent>
             <div className="ChartListItem__content">
-              <h3 className="ChartListItem__content__title">{f.metadata.name}</h3>
-              <div className="ChartListItem__content__info text-r">
-                <span
-                  className={`ChartListItem__content__repo padding-tiny
-                  padding-h-normal type-small margin-t-small`}
+              <div className="ChartListItem__content__title type-big">{f.metadata.name}</div>
```
Comment (Angelmmiguel) [inline dashboard/src/components/FunctionList/FunctionListItem.tsx]: Same here

**Comment by sameersbn** [comment]
updated!

**Comment by sameersbn** [comment]
sorry.. need to update the paragraph thingy.

**Comment by sameersbn** [comment]
okay! please take a look. thanks @Angelmmiguel

File: dashboard/src/components/ChartList/ChartListItem.css [hunk 6]
```diff
@@ -3,34 +3,34 @@
...
-  display: flex;
-  justify-content: space-between;
 }
 
 .ChartListItem__content__title {
-  /* Avoid 3 line elements */
-  word-break: break-all;
-  flex: 1;
-  font-size: 1.1em;
+  color: black;
```
Comment (Angelmmiguel) [inline dashboard/src/components/ChartList/ChartListItem.css]: Could you update this color to the text one? `#1C2B39` https://design.bitnami.com/category/Foundations/Variables/index.html#Colors

**Pull request merged**